server/src-lib/Hasura/RQL/DML/Select/Internal.hs

```diff
@@ -153,6 +152,13 @@ buildJsonObject pfx parAls arrRelCtx strfyNum flds =
                      ANIField (fldAls, arrSel)
         in S.mkQIdenExp arrPfx fldAls
 
+    toSQLCol :: PGColInfo -> Maybe ColOp -> S.SQLExp
+    toSQLCol col colOpM = toJSONableExp strfyNum (pgiType col) $ case colOpM of
```

Comment:
Move code from `toJSONable...` to new line. In that way code looks pretty :smile: